index.js

```diff
@@ -191,6 +191,28 @@ export default class Supercluster {
         return expansionZoom;
     }
 
+    getPointUnclusterZoom(point) {
+        const [lng, lat] = point;
+        const pointX = fround(lngX(lng));
+        const pointY = fround(latY(lat));
+
+        let expansionZoom = this.options.minZoom;
+        while (expansionZoom < this.options.maxZoom) {
```

Comment:
Since points are more likely to get clustered on higher zooms, it would likely be faster to iterate starting from `maxZoom` and up until there are no points in the location or are all clusters.